tests/testthat/helper-expect.r

```diff
@@ -1,7 +1,7 @@
 # Set up local module environment to test against.
 # Note that we override the normal path here.
 
-mod::set_options(path = 'modules')
+set_options(path = 'modules')
```

Comment:
This should be fully qualified:

```suggestion
mod::set_options(path = 'modules')
```

Reply:
but it doesn't work for `make test` if package is not installed?

Reply:
still doesn't work with [`reexport`](https://github.com/klmr/modules/pull/143)